v2/devices/FP4.yml

```diff
@@ -196,3 +196,41 @@ operating_systems:
           - core:user_action:
               action: "recovery"
     slideshow: []
+
+  - name: "postmarketOS"
+    compatible_installer: ">=0.9.7-beta"
```

Comment:
`compatible_installer: ">=0.9.8-beta"`

Because `0.9.7` has been released since then and the required `codename` changes will be in with `0.9.8`.